resources/serverstatus.ts

```diff
@@ -35,7 +35,7 @@ const ec2Client = new EC2Client({ region: REGION });
 /**
  * Everything here has the assumption there is only one task.
  */
-const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
+export const handler: APIGatewayProxyHandler = async (event: APIGatewayEvent, context: Context) => {
```

Comment:
Lets specify the return type to clarify the required response values: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L140-L154